mmv1/third_party/terraform/tests/resource_cloud_identity_group_membership_test.go.erb

```diff
@@ -11,6 +10,165 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 )
 
+func TestAccCloudIdentityGroupMembership_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_domain":    getTestOrgDomainFromEnv(t),
+		"cust_id":       getTestCustIdFromEnv(t),
+		"identity_user": getTestIdentityUserFromEnv(t),
+		"random_suffix": randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckCloudIdentityGroupMembershipDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCloudIdentityGroupMembership_update1(context),
```

Comment:
Can you add an import step after each config step? They're a good way to ensure that each step is stable.

Reply:
✅ 30bc07ed3a7f8ac2e7f6ae0e436d3ece5f0ef25d